agent/www/components/addresses/addresses_ctrl.js

```diff
@@ -218,26 +218,33 @@ angular.module('patternfly.toolbars').controller('ViewCtrl', ['$scope', '$timeou
           }
         };
 
-        function is_valid_filter(filter) {
-            if (filter.title === 'Name') {
-                try {
-                    $scope.notification.show_alert = false;
+        var filterChange = function (filters) {
+            $scope.filtersText = "";
 
-                    var dummy = new RegExp(filter.value);
+            $scope.notification.show_alert = false;
+            var messages = [];
+            var valid = filters.filter(function (filter) {
+                if (filter.title === 'Name') {
+                    try {
+                        $scope.notification.show_alert = false;
+
+                        var dummy = new RegExp(filter.value);
+                        return true;
+                    } catch (error) {
+                        $scope.notification.show_alert = true;
+                        $scope.notification.alert_msg = '"' + filter.value + '": is not valid.  Only regular expressions are supported.';
```

Comment:
Should we be using messages, instead of $scope.notification.alert_msg?   As it stands now, line 243 messages.length > 0, will always be false, since messages isn't being populated.